scenes/passy_scene_read.c

```diff
@@ -95,18 +95,25 @@ bool passy_scene_read_on_event(void* context, SceneManagerEvent event) {
             if(passy->bytes_total == 0) {
                 popup_set_header(popup, "Reading", 68, 30, AlignLeft, AlignTop);
             } else {
-                // Update the header with the current bytes read
-                char header[32];
-                snprintf(
-                    header,
-                    sizeof(header),
-                    "Reading\n%d/%dk",
-                    passy->offset,
-                    (passy->bytes_total / 1024));
-                popup_set_header(popup, header, 68, 30, AlignLeft, AlignTop);
+                //TODO: fix still not showing progress
```

Comment:
This is another thing that seems unrelated to the PACE update and can be it's own PR.  It would also be nice as it's own PR so it could explain some explanation for the reason for the change (bug?  feature?)